### DISCUSSION TITLE
Example of how does SEARCH work in Razor page?

### DISCUSSION THREAD

**Discussion by orchardc** [OrchardCMS/OrchardCore#8000] [General]
I created some contents in OC, how to utilise the Search function to search those contents?

There are some examples in liquid page, but how about Razor page, do we have workable example?

Many thanks!!

**Comment by hishamco** [comment]
Have a look to https://docs.orchardcore.net/en/dev/docs/guides/implement-fulltext-search/ it should works for both Liquid or cshtml

**Comment by orchardc** [comment]
Hi @hishamco , thanks for the info, but I'm looking for example, in a Razor page how to call OC Lucene service and get the list of result back? 

Hope it's clear.

**Reply by Skrypt** [reply]
In the OrchardCore.Lucene module there is a SearchController that can be used as a starting point.
